src/SwedbankPay.Sdk/PaymentInstruments/Swish/SwishPaymentOptions.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace SwedbankPay.Sdk.PaymentInstruments.Swish
+{
+    public class SwishPaymentOptions
```

Comment:
This makes me unsure. I like the use of `Options`, but can't think of a good pairing name for the corresponding `Payment` class.

```suggestion
    public class SwishRequestData
```